### ISSUE TITLE
Line been cutoff in flow chart

### ISSUE THREAD

**Issue by jackycute** [mermaid-js/mermaid#1288]
**Describe the bug**
Lines been cutoff in flow chart
Given a simple flow chart graph:
```
graph TD
    a --> c
    a --> e
    d --> e
    c --> e
    c --> d
```

will result in:

<img width="689" alt="螢幕快照 2020-03-02 下午5 41 26" src="https://user-images.githubusercontent.com/2639151/75664221-146ce580-5cad-11ea-9352-89782b799fa4.png">

the line connects a and e nodes should not been cutoff in the output.

**To Reproduce**
1. Go to https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggVERcbiAgICBhIC0tPiBjXG4gICAgYSAtLT4gZVxuICAgIGQgLS0-IGVcbiAgICBjIC0tPiBlXG4gICAgYyAtLT4gZCIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In19

**Expected behavior**
the line connects a and e nodes should not been cutoff in the output.

**Screenshots**
<img width="689" alt="螢幕快照 2020-03-02 下午5 41 26" src="https://user-images.githubusercontent.com/2639151/75664221-146ce580-5cad-11ea-9352-89782b799fa4.png">

**Desktop (please complete the following information):**
 - OS: macOS 10.13.6
 - Browser: Chrome 79
 - Version: 8.4.8

**Smartphone (please complete the following information):**

**Additional context**

**Comment by dcadc** [comment]
Looks like the place where `a --> e` was defined matters.
![image](https://user-images.githubusercontent.com/8225179/75666039-16847380-5cb0-11ea-826c-887f78002d99.png)
![image](https://user-images.githubusercontent.com/8225179/75665982-fd7bc280-5caf-11ea-8e28-38424616b03f.png)

**Comment by GDFaber** [comment]
This has been fixed in #1333 so we can close here.
